### PR TITLE
[RFR] Stabilize e2e tests

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

**Comment by fzaninotto** [comment]
test fail 🙀

**Comment by djhi** [comment]
I know, they randomly fail. Moreover, the failing tests are random too. Sometime that's the auth tests, sometimes the filter tests

**Comment by djhi** [comment]
Should be fine now: I restarted the travis build ten times in a row

File: e2e/pages/CreatePage.js [hunk 1]
```diff
@@ -19,9 +19,10 @@ module.exports = (url) => (driver) => ({
 
     waitUntilDataLoaded() {
         let continued = true;
-        return driver.wait(until.elementLocated(this.elements.appLoader), 400)
+        return driver.wait(until.elementLocated(this.elements.appLoader), 2000)
             .catch(() => continued = false) // no loader - we're on the same page !
             .then(() => continued ? driver.wait(until.stalenessOf(driver.findElement(this.elements.appLoader))) : true)
+            .catch(() => continued = false) // no loader - we're on the same page !
```
Comment (fzaninotto) [inline e2e/pages/CreatePage.js]: What's the use of adding a `continues = false` here if no promise looks at this variable later in the chain?

File: e2e/tests/auth.js [hunk 2]
```diff
@@ -9,24 +10,20 @@ describe('Authentication', () => {
     it('should go to login page after logout', async () => {
         await ListPage.navigate();
         await ListPage.logout();
-        await driver.sleep(300);
-        assert.equal(await driver.getCurrentUrl(), 'http://localhost:8083/#/login');
+        await driver.wait(until.urlIs('http://localhost:8083/#/login'));
```
Comment (fzaninotto) [inline e2e/tests/auth.js]: no more assert... not good!

File: e2e/pages/CreatePage.js (hunk 1 above)
Comment (djhi) [inline e2e/pages/CreatePage.js]: None, what matters is catching the error and continue

File: e2e/tests/auth.js (hunk 2 above)
Comment (djhi) [inline e2e/tests/auth.js]: I can put back the assertion after the new wait. Would that be good ?

Comment (djhi) [inline e2e/tests/auth.js]: Or I can add a specific message instead of the default timeout message

**Pull request merged**